src/ari_sxn_simbeamline/xrt_sim.py

```diff
@@ -0,0 +1,131 @@
+import sys
```

Comment:
The standard Jiemin is for 'import' statements to be listed in alphabetical order